Sources/Swift/FlexLayout.swift

```diff
@@ -1213,6 +1213,46 @@ public final class Flex {
         return self
     }
     
+    //
+    // MARK: Gap
+    //
+    
+    /**
+     Get setted value through func `setGap(_ gutter: Gutter, _ value: CGFloat)`.
+     
+     - Parameter gutter: directions
+     - Returns: distance
+    */
+    @discardableResult
+    public func getGap(_ gutter: Gutter) -> CGFloat {
+        switch gutter {
+        case .column:   return yoga.columnGap
+        case .row:      return yoga.rowGap
+        case .all:      return yoga.gap
+        }
+    }
+
+    /**
+     Set distance between rows and columns.
+     
+     - Parameters:
+       - gutter: directions
+       - value: distance
+     - Returns: flex interface
+    */
+    @discardableResult
+    public func setGap(_ gutter: Gutter, _ value: CGFloat) -> Flex {
+        switch gutter {
+        case .column:               
+            yoga.columnGap = value
+        case .row:                  
+            yoga.rowGap = value
+        case .all:
+            yoga.gap = value
+        }
+        return self
+    }
```

Comment:
Is there a reason to provide getter for gap?
All properties of the existing flex are provided only with setters.
so I think it can cause confusion in the interface.

Additionally, i propose a direction that provides the each gap function separately.

```swift
func columnGap(_ value: CGFloat) -> Flex
func rowGap(_ value: CGFloat) -> Flex
func gap(_ value: CGFloat) -> Flex
```

Reply:
I wanted the code works similar like [Flexbox's function](https://github.com/facebook/yoga/releases/tag/v2.0.0).
But it looks better. Thank you.

The reason that I provided getter for gap was just a idea that 'Maybe there someone who needs it' like this [case](https://stackoverflow.com/questions/68824785/how-to-get-flexbox-item-width-dynamically-including-the-gap-value)
And I also think it doesn't fit with the other code. So we removed it